perses/tests/test_repex.py

```diff
@@ -39,6 +37,7 @@ def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
                 "1",
                 "2",
                 mutant_name.upper(),
+                padding=1.7 * unit.nanometers,
```

Comment:
Why is that we need to increase the default padding here? Thinking that would make the test to take more time to run.

Reply:
I decreased it back down and left a TODO to increase it once we move to testing against openmm >= 7.8